examples/05_Collections/14_sorted.md

```diff
@@ -10,6 +10,8 @@ import kotlin.math.abs
 fun main() {
 
 //sampleStart
+    import java.lang.Math.abs
```

Comment:
The appropriate import is already in the first line of the example, please remove this line.

Reply:
awesome, will do on both the naming and removing the import.